Feature/admin

pritig-cuelogic opened the pull request:
Hi,

Created admin user and customize admin form.

pritig-cuelogic committed:
change

pritig-cuelogic committed:
change

ananthu-nair707 commented on mysite/polls/admin.py:
```diff
@@ -1,3 +1,18 @@
+	fieldsets = [
+	    (None,               {'fields': ['question_text']}),
+	    ('Date information', {'fields': ['pub_date'], 'classes': ['collapse']}),
+	]
```
Why long space after None? 
Check PEP8
https://www.python.org/dev/peps/pep-0008/#pet-peeves

ananthu-nair707 commented on mysite/polls/views.py:
```diff
@@ -23,9 +24,20 @@ def detail(request, question_id):
+	else:
+		selected_choice.votes += 1
+		selected_choice.save()
+		return HttpResponseRedirect(reverse('polls:results', args=(question.id,)))
```
Use a ruler of 80 characters if you are using sublime.
80 char rule PEP8

ananthu-nair707 reviewed (changes requested):
Please make required changes.

pritig-cuelogic committed:
Merge branch 'dev' of https://github.com/pritig-cuelogic/Django into feature/admin

pritig-cuelogic committed:
Changes in following file.
1. remove space after none in mysite/polls/admin.py
2.80 char rule applied rest of files.
